metadata-service/restli-client/src/main/java/com/linkedin/entity/client/RestliEntityClient.java

```diff
@@ -456,7 +456,7 @@ public SearchResult searchAcrossEntities(@Nonnull List<String> entities, @Nonnul
   @Nonnull
   public SearchResult searchAcrossEntities(@Nonnull List<String> entities, @Nonnull String input,
       @Nullable Filter filter, int start, int count, @Nullable SearchFlags searchFlags,
-      @Nonnull final Authentication authentication, @Nullable List<String> facets)
+      @Nullable SortCriterion sortCriterion, @Nonnull final Authentication authentication, @Nullable List<String> facets)
       throws RemoteInvocationException {
 
     final EntitiesDoSearchAcrossEntitiesRequestBuilder requestBuilder =
```

Comment:
I think the `EntitiesDoSearchAcrossEntitiesRequestBuilder` actually has a parameter for the sort criterion: Do you want to add it here as well?

Reply:
nice good call! adding now